buildSrc/src/main/groovy/csde.gradle

```
@@ -154,7 +154,8 @@ tasks.register("stopAndCleanCorda") {
     doLast {
         try {
             cordaLifeCycle.stopCorda()
-        } catch (Exception ignored) {
+        } catch (Exception ignore) {
+            println(ignore.toString())
```

I think you need to state that you're ignoring to your fellow developer who will debug this in the future:
```suggestion
            println("ignoring exception: ${ignore.toString()}")
```

---

done